contracts/state/State.sol

```diff
@@ -144,8 +154,10 @@ contract State is Ownable2StepUpgradeable, IState {
         uint256 methodId,
         bytes calldata methodParams
     ) public {
+        bytes2 idType = GenesisUtils.getIdType(id);
+        require(_stateData.isIdTypeSupported[idType], "id type is not supported");
```

Comment:
Use the modifier instead

Reply:
@AndriianChestnykh not sure that use modifier is reasonable here. In any case I need to calculate `idType` to use it below in the code. Why do I need calculate `idType` it twice?

Reply:
ah, sorry I missed that